config/gen/bot.py

```diff
@@ -60,6 +60,11 @@
 if "jaia_data_offload_ignore_type" in os.environ:
```

Comment:
Hey Nick can you start this branch from 1.y so it does not mark the code from the branch you built from as new. Also can you make the B in the pr name lowercase